data/mapcss-mapping.csv

```diff
@@ -1321,8 +1321,8 @@ historic|memorial|statue;[historic=memorial][memorial=statue],[historic=memorial
 historic|memorial|plaque;[historic=memorial][memorial=plaque],[historic=memorial][memorial:type=plaque];;name;int_name;1233;
 historic|castle|defensive;[historic=castle][castle_type=defensive];;name;int_name;1234;
 historic|castle|stately;[historic=castle][castle_type=stately];;name;int_name;1235;
-tourism|attraction|animal;[tourism=attraction][attraction=animal];;name;int_name;1236;
-tourism|attraction|specified;[tourism=attraction][attraction=specified];;name;int_name;1237;
+attraction|animal;[attraction=animal];;name;int_name;1236;
+deprecated:tourism|attraction|specified:03.2024;[tourism=attraction][attraction=specified];x;name;int_name;1237;x
```

Comment:
```;x;``` in the middle means that types is removed - ok
```;x``` in the end is an error. Put replacement type if any, or keep empty like
```deprecated:tourism|attraction|specified:03.2024;[tourism=attraction][attraction=specified];x```